modules/mesh-layers/src/simple-mesh-layer/simple-mesh-layer-vertex.glsl.js

```diff
@@ -32,7 +34,9 @@ void main(void) {
 
   vTexCoord = texCoords;
   vColor = instanceColors;
-  // vLightWeight = lighting_getLightWeight(position_commonspace.xyz, project_normal(normals));
+  vec3 normals_commonspace = project_normal(instanceModelMatrix * normals);
+  lightColor = lighting_getLightColor(instanceColors.rgb, project_uCameraPosition, position_commonspace.xyz, normals_commonspace);
+  specularColor = lighting_getSpecularLightColor(project_uCameraPosition, position_commonspace.xyz, normals_commonspace);
```

Comment:
You'll get discontinuities if you calculate specular highlights in the vertex shader.